sdk/servicebus/service-bus/samples/typescript/gettingStarted/session.ts

```diff
@@ -89,7 +89,7 @@ async function receiveMessages(ns: ServiceBusClient, sessionId: string): Promise
   const onMessage = async (brokeredMessage: ServiceBusMessage) => {
     console.log(`Received: ${brokeredMessage.sessionId} - ${brokeredMessage.body} `);
   };
-  const onError: OnError = (err) => {
+  const onError: OnError = (err): void => {
```

Comment:
Did you ever figure out why the rule is requiring this change even with `allowTypedFunctionExpressions: true` ?

Reply:
no i am not sure why it's still generating that error - this just happened in 1-2 places though.